identity/app/views/fragments/profile/accountDetailsForm.scala.html

```diff
@@ -3,7 +3,7 @@
 @import views.html.fragments.form.{inputField, textareaField}
 @import form.IdFormHelpers._
 @import form.Input
-@import model.Countries
+@import model.{Countries, CountryCodes}
 @import org.joda.time.DateTime.now
 @import helper._
 
```

Comment:
@susiecoleman is right, but there is a way to do it without adding a 3rd submit button, and without JS. Requires fiddling more with CSS though.

- Move the "Save Changes" button to the top of the form, then make the `<form>` element have CSS `position: relative`, and then absolutely position the button at the bottom of the form with:
```css
button {
  position: absolute;
  bottom: 0;
}
```
- Move the "Delete Phone Number" button to the bottom of the form, then set its position manually within the form (not very scalable as would need to be a constant).

Simplest solution using JS is to make the "Delete Phone Number" button a normal button (not a submit), then add the `type="submit" on click: (should work, but untested)
```html
<button name="thing" onclick="this.type='submit'">Delete Phone Number</button>
```